### DISCUSSION TITLE
Release 3.0.1

### DISCUSSION THREAD

**Discussion by ChenglongMa** [ChenglongMa/zoplicate#35] [General]
## What's Changed

In this version, we have made the following changes:

1. 🧬 **CHANGE!**: We have updated the UI of buttons in the duplicate pane.
2. 🐛 **FIX!**: We have optimized the performance of duplicate search and detection.

* Release v3.0.1 by @ChenglongMa in https://github.com/ChenglongMa/zoplicate/pull/34

**Full Changelog**: https://github.com/ChenglongMa/zoplicate/compare/3.0.0...3.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ChenglongMa/zoplicate/releases/tag/3.0.1'>Release 3.0.1</a>.</em>

**Comment by richardcmckinney** [comment]
Pardon my ignorance here, but any insights on why this plugin wouldn't be working with Zotero running on Linux?

**Reply by ChenglongMa** [reply]
Hi @richardcmckinney,

Thanks for your attention!

May I know what Zotero version you are using? Zoplicate only works in [Zotero 7 (beta)](https://www.zotero.org/support/beta_builds).

If it still does not work, please check the `Error console` for any error messages about Zoplicate:

![Snipaste_2024-05-05_09-11-31](https://github.com/ChenglongMa/zoplicate/assets/33501679/f7976cf0-5763-4f6f-9573-41470a58216e)

I will also test it on a Linux device.

Thanks for your patience!

**Reply by ChenglongMa** [reply]
Hi @richardcmckinney,

I tested Zotero 7 and Zoplicate 3.0.1 on a Ubuntu 22.04 virtual machine.
And they are working well:

![Snipaste_2024-05-05_10-52-03](https://github.com/ChenglongMa/zoplicate/assets/33501679/167725b2-4c56-42cb-91bb-c0813fee8caf)

Please let me know if you have any findings.

Thanks!
